### ISSUE TITLE
[Request] Plugin Blacklist needs individual Descriptions

### ISSUE THREAD

**Issue by sbts** [foosel/OctoPrint#2313]
#### What were you doing?

Currently the blacklist only lists the plugin name and relevant version numbers.

#### What did you expect to happen?

A brief description of why the plugin is blacklisted should be present.
This should include any relevant issue numbers or links to other discussion.

The reason I'm suggesting this, is as the list grows longer (hopefully it doesn't....), it will help both users and plugin maintainers know why blacklisting has happened.
That will allow a blacklist entry to be added immediately, with no specific notification issued to the maintainer in advance

**Comment by GitIssueBot** [comment]
Hi @sbts,

It looks like there is some **information missing** from your bug report that will be needed in order to solve the problem. Read the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md) which will provide you with a template to fill out here so that your bug report is ready to be investigated (I promise I'll go away then too!).

If you did not intend to report a bug but wanted to **request a feature or brain storm** about some kind of development, please take special note of the title format to use as described in the [Contribution Guidelines](https://github.com/foosel/OctoPrint/blob/master/CONTRIBUTING.md).

**Please do not abuse the bug tracker as a support forum** - if you have a question or otherwise need some kind of help or support refer to the [Mailinglist](https://groups.google.com/group/octoprint) or the [G+ Community](https://plus.google.com/communities/102771308349328485741) instead of here.

Also **make sure you are at the right place** - this is the bug tracker of the official version of OctoPrint, not the Raspberry Pi image OctoPi nor any unbundled third party OctoPrint plugins or unofficial versions. Make sure too that you have **read through the [Frequently Asked Questions](https://github.com/foosel/OctoPrint/wiki/FAQ)** and searched the [**existing tickets**](https://github.com/foosel/OctoPrint/search?q=&ref=cmdform&type=Issues) for your problem - try multiple search terms please.

I'm marking this one now as needing some more information. Please understand that if you do not provide that information within the next two weeks (until 2018-01-06 07:00 UTC) I'll close this ticket so it doesn't clutter the bug tracker. This is nothing personal, so please just be considerate and help the maintainers solve this problem quickly by following the guidelines linked above. Remember, the less time the devs have to spend running after information on tickets, the more time they have to actually solve problems and add awesome new features. Thank you!

Best regards,
~ Your friendly GitIssueBot

PS: I'm just an automated script, not a human being, so don't expect any replies from me :) Your ticket is read by humans too, I'm just not one of them.

**Comment by sbts** [comment]
I'm hoping the typo in the `[Request]` tag in the Subject was the cause of the incomplete ticket trigger.
It would really help if GitissueBot indicated to the user what information was missing etc

**Comment by foosel** [comment]
I initially thought about this as well but then figured the blacklist would hopefully not see that much use (it IS for emergencies only). I agree with your points though, and it should be an easy addition. Best of all - no need to adjust OctoPrint for it, only the [blacklist page on plugins.octoprint.org](https://plugins.octoprint.org/blacklist/), so this could be implemented quickly "in between".

I've added an issue for that in the plugin repository tracker (better location since it's what would need to be adjusted, not OctoPrint), see OctoPrint/plugins.octoprint.org#151. Closing this ticket and proceeding over there.
